functions/src/index.test.ts

```diff
@@ -0,0 +1,137 @@
+// @firebase/testing module is used to interact with the emulator that runs locally.
+// It's mostly described in the context of Firestore rules testing: https://firebase.google.com/docs/rules/unit-tests
+import * as firebaseTesting from "@firebase/testing";
+import * as functions from "firebase-functions-test";
+// This is necessary automatically setup config for test environment.
+// If you want to connect to real Firestore, configuration can be provided here.
+// Otherwise, emulator is needed. Emulator is set using env variable: FIRESTORE_EMULATOR_HOST
+// (check package.json scripts).
+const projectId = "test-project";
+const firebaseFunctionsEnv = functions({ projectId });
+// Require functions AFTER calling functions.
```

Comment:
To clarify here: `webApiV1` is the "functions" that are being required after the firebaseFunctionEnv is setup is that a correct interpretation of this comment?

Reply:
Yes. Right, this comment wasn't very clear. I tried to improve naming and these lines in the last commit.